src/vivarium_nih_us_cvd/constants/data_values.py

```diff
@@ -361,9 +363,9 @@ def name(self):
 class __MedicationAdherenceType(NamedTuple):
     """medication adherence types"""
 
-    ADHERENT: str = "adherent"
-    PRIMARY_NON_ADHERENT: str = "primary_non_adherent"
-    SECONDARY_NON_ADHERENT: str = "secondary_non_adherent"
+    PRIMARY_NON_ADHERENT: str = "cat1"
```

Comment:
Is it standard practice to keep the categories as "cat1", "cat2", etc instead of more descriptive? And then if you need to stratify results by them perhaps map them to the more descriptive labels?

Reply:
I believe it is standard practice in line with the GBD data scheme.